src/mpl/src/misc/Makefile.mk

```diff
@@ -4,4 +4,5 @@
 #     See COPYRIGHT in top-level directory.
 #
 
-lib@MPLLIBNAME@_la_SOURCES += src/misc/mpl_misc.c
+lib@MPLLIBNAME@_la_SOURCES += src/misc/mpl_misc.c \
+                              src/misc/mpl_mkstemp.c
```

Comment:
Did you forget to add `src/misc/mpl_mkstemp.c`?

Reply:
indeed i did. added now.